cli/post-install.js

```diff
@@ -0,0 +1,66 @@
+import { executeCommand } from './utils.js';
+
+const OS = await executeCommand('uname -s', true);
+const ARCH = await executeCommand('uname -m', true);
+
+if (OS !== 'Linux' && OS !== 'Darwin') {
```

Comment:
I'm not very sure on this: windows doesn't have a uname command. If so, we want to try catch above commands to catch windows.